eo-maven-plugin/src/test/java/org/eolang/maven/ParseMojoTest.java

```diff
@@ -117,29 +117,12 @@ void parsesWithCache(@TempDir final Path temp) throws Exception {
 
     @Test
     void crashesOnInvalidSyntax(@TempDir final Path temp) {
```

Comment:
@c71n93 I think the test does not belong to the class. Here we should test `ParseMojo`, not `VerifyMojo`. Let's move it `VerifyMojoTest`. But here I believe we should add the test (if it does not exist yet) that would check that invalid program DOES NOT fail

Reply:
@maxonfjvipon done